R/set_tech_trajectories.R

```diff
@@ -22,26 +22,35 @@
 #' @param scenario_to_follow_baseline Character. A string that indicates which
 #'   of the scenarios included in the analysis should be used to set the
 #'   baseline technology trajectories.
+#' @param emission_factors boolean. also summarise emmision factors? only needed
+#'   for lititgation risk at this point.
```

Comment:
same as above, if this can be around for trisk I would suggest not doing the distinguiation

Reply:
removed distinction